mayhem/Mayhemfile

```diff
@@ -0,0 +1,12 @@
+# Namespaced project name that the target belongs to
```

Comment:
Let's try not to call our target `fuzz.libfuzzer`. Perhaps something more descriptive?